src/gui/transferlistwidget.cpp

```diff
@@ -649,7 +649,13 @@ void TransferListWidget::recheckSelectedTorrents()
 void TransferListWidget::reannounceSelectedTorrents()
 {
     for (BitTorrent::Torrent *const torrent : asConst(getSelectedTorrents()))
-        torrent->forceReannounce();
+    {
+        if (!torrent->isStopped())
```

Comment:
Aren't such actions silently ignored by stopped torrents?

Reply:
I haven't looked at the libtorrent codebase to confirm. I noticed that `TrackerListWidget` performed this check but other places didn't, so I thought it would make sense to standardize it.

Reply:
> I haven't looked at the libtorrent codebase to confirm.

It would be strange (actually, it would be incorrect) if stopped torrents perform announces. So `libtorrent` doesn't do that (yes, I've confirmed it). So we can standardize it in opposite way (i.e. not checking for "stopped").

Reply:
It appears libtorrent has a similar [check for DHT announces](https://github.com/arvidn/libtorrent/blob/98e02bf5dadccb2ba25b9d74f89aa50732f740e4/src/torrent.cpp#L700) so I will remove this condition.